### ISSUE TITLE
Could not load file or assembly 'Avalonia.Visuals

### ISSUE THREAD

**Issue by j4587698** [kikipoulet/SukiUI#11]
1. `dotnet new -i Avalonia.Templates` install Template
2. update all nuget package to latest(now is 11-preview4)
3. install SukiUi latest(now is 4.1.0)
4. add ` <StyleInclude Source="avares://SukiUI/Theme/Index.xaml"/>` to `App.xaml`
5. run 

It will throw 
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IO.FileNotFoundException: Could not load file or assembly 'Avalonia.Visuals, Version=0.10.13.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b'. 
File name: 'Avalonia.Visuals, Version=0.10.13.0, Culture=neutral, PublicKeyToken=c8d484a7012f9a8b'
   at CompiledAvaloniaXaml.!AvaloniaResources.Populate:/Theme/DialogHostStyle.axaml(IServiceProvider , Styles )
   at CompiledAvaloniaXaml.!AvaloniaResources.Build:/Theme/DialogHostStyle.axaml(IServiceProvider )
   at CompiledAvaloniaXaml.!XamlLoader.TryLoad(String )
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at Avalonia.Markup.Xaml.AvaloniaXamlLoader.Load(Uri uri, Uri baseUri) in /_/src/Markup/Avalonia.Markup.Xaml/AvaloniaXamlLoader.cs:line 57
   at Avalonia.Markup.Xaml.Styling.StyleInclude.get_Loaded() in /_/src/Markup/Avalonia.Markup.Xaml/Styling/StyleInclude.cs:line 54
   at Avalonia.Markup.Xaml.Styling.StyleInclude.Avalonia.Controls.IResourceProvider.AddOwner(IResourceHost owner) in /_/src/Markup/Avalonia.Markup.Xaml/Styling/StyleInclude.cs:line 98
   at Avalonia.Styling.Styles.Avalonia.Controls.IResourceProvider.AddOwner(IResourceHost owner) in /_/src/Avalonia.Base/Styling/Styles.cs:line 212
   at Avalonia.Markup.Xaml.Styling.StyleInclude.Avalonia.Controls.IResourceProvider.AddOwner(IResourceHost owner) in /_/src/Markup/Avalonia.Markup.Xaml/Styling/StyleInclude.cs:line 98
   at Avalonia.Styling.Styles.InternalAdd(IList items, IResourceHost owner, StyleCache& cache) in /_/src/Avalonia.Base/Styling/Styles.cs:line 257
   at Avalonia.Styling.Styles.OnCollectionChanged(Object sender, NotifyCollectionChangedEventArgs e) in /_/src/Avalonia.Base/Styling/Styles.cs:line 303
   at Avalonia.Collections.AvaloniaList`1.NotifyAdd(T item, Int32 index) in /_/src/Avalonia.Base/Collections/AvaloniaList.cs:line 700
   at Avalonia.Collections.AvaloniaList`1.Add(T item) in /_/src/Avalonia.Base/Collections/AvaloniaList.cs:line 205
   at Avalonia.Styling.Styles.Add(IStyle item) in /_/src/Avalonia.Base/Styling/Styles.cs:line 173
   at testsuki.App.!XamlIlPopulate(IServiceProvider , App ) in D:\dotnet\testsuki\App.axaml:line 5
   at testsuki.App.!XamlIlPopulateTrampoline(App ) in D:\dotnet\testsuki\App.axaml:line 0
   at testsuki.App.Initialize() in D:\dotnet\testsuki\App.axaml.cs:line 11
   at Avalonia.Controls.AppBuilderBase`1.Setup() in /_/src/Avalonia.Controls/AppBuilderBase.cs:line 287
   at Avalonia.Controls.AppBuilderBase`1.SetupWithLifetime(IApplicationLifetime lifetime) in /_/src/Avalonia.Controls/AppBuilderBase.cs:line 148
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime[T](T builder, String[] args, ShutdownMode shutdownMode) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 208
   at testsuki.Program.Main(String[] args) in D:\dotnet\testsuki\Program.cs:line 12
```

**Comment by kikipoulet** [comment]
Well, we just found a kind of rabbit hole.
I already had this exact error while migrating to 0.11.0.

I just realized that the version of Avalonia 0.11.0 SukiUI is using is from a weird AvaloniaCI myget.org source, not a nuget.org one. Now I'm trying to move to the nuget.org version, but weirdly it breaks lots of things. Weird.

So I move the project to the "official" Avalonia Nuget source, work on broken things and hope it will be okay in few days.
Thanks for the warning :)

**Comment by kikipoulet** [comment]
Okay kind of good news here. 
Problems seems to be solved, it was a mix of control's template name change between Avalonia 0.10 and Avalonia 0.11, dependencies from different nuget source (don't know how it even happened !) and different dependencies with DialogHost. Got some nice headache here.

Well, I brought SukiUI to the nuget.org Preview1 package, and cleaned all the dependencies to the official.

Two problems stays : 

- I can't run the android app on Preview4 but only on Preview1. Problem is : ToggleSwitch Knob Name is different in each version. So the toggleswitch works only from Preview4 now.

- Lost some of my dialoghost work, to align the dialog in the bottom for mobile controls, when migrating. Will make a feature request in the DialogHost repository to stay with the "official" version and not create a shitty fork again. 

Now I'm working on testing the "easy" installation on new avalonia 0.11 project normal and Mobile, and will update the package + installation instructions when everything will be fine. I'm a bit relieved by the way 👍

**Comment by kikipoulet** [comment]
Solved with 4.2.1 nuget package, see the installation instructions in the wiki
